packages/web-client/ember-cli-build.js

```diff
@@ -3,8 +3,37 @@
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const path = require('path');
 const concat = require('broccoli-concat');
+const compileCSS = require('broccoli-postcss');
+const postcss = require('postcss');
 const webpack = require('webpack');
 
+const prependSelector = postcss.plugin(
+  'postcss-prepend-selector',
+  function (opts) {
+    opts = opts || {};
+    return function (css) {
+      css.walkRules(function (rule) {
+        rule.selectors = rule.selectors.map(function (selector) {
+          if (/(^|\b)(html|:root)\b/.test(selector)) {
+            return selector;
+          }
```

Comment:
The `#web-client-root` selector is on the html element, so we don't want to do anything with selectors that are targeting html or root.